index.html

```diff
@@ -666,6 +678,45 @@ <h2 id="AudioContext">
           cases, only a single <a><code>AudioContext</code></a> is used per
           document.
         </p>
+        <p>
+          <!-- ReSpec does not support documenting constructor -->
```

Comment:
Actually, ReSpec does support constructor docs although it's a bit weird. Use this approach inside a <dl class="idl"> element for the interface:
     <dt>Constructor</dt>
     <dd>
       <dl class="parameters">
        ...
    </dd>